htdocs/comm/action/peruser.php

```diff
@@ -489,6 +490,9 @@
 	$newcardbutton .= dolGetButtonTitle($langs->trans("AddAction"), '', 'fa fa-plus-circle', DOL_URL_ROOT.'/comm/action/card.php?action=create&datep='.sprintf("%04d%02d%02d", $tmpforcreatebutton['year'], $tmpforcreatebutton['mon'], $tmpforcreatebutton['mday']).$hourminsec.'&backtopage='.urlencode($_SERVER["PHP_SELF"].($newparam ? '?'.$newparam : '')));
 }
 
+if (empty($num)) {
```

Comment:
Juste set $num = '';
We are in a global page so if you can't find the var defined before, it means it is never set so we can just init it here.

Reply:
Ok, modified it too